### DISCUSSION TITLE
线上Mq重启后重复消费了

### DISCUSSION THREAD

**Discussion by hades-ruan** [apache/rocketmq#7443] [Q&A]
RocketMq 5.0.0 版本，线上MQ修改了配置重启后，发现延时队列中的消息重新被消费了

**Comment by figuewang** [comment]
消费者设置 CONSUME_FROM_LAST_OFFSET，第一次启动从队列的最后位置开始消费

**Comment by gaoyf** [comment]
这个问题已经修复了，#6881， 版本>=5.1.2即可
